sql/updates/world/2016_04_24_body_of_evidence_fixes.sql

```diff
@@ -0,0 +1,8 @@
+-- Add missing Boulderfist Saboteur lines for quest Body of Evidence (Quest 9932)
+INSERT INTO script_texts (entry, content_default, content_loc1, content_loc2, content_loc3, content_loc4, content_loc5, content_loc6, content_loc7, content_loc8, sound, type, language, emote, comment) VALUES (-1900200, 'Shadow Council come for you Warmaul! We is mad at you!', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 0, 0, 0, 0, 'Boulderfist Saboteur');
+INSERT INTO script_texts (entry, content_default, content_loc1, content_loc2, content_loc3, content_loc4, content_loc5, content_loc6, content_loc7, content_loc8, sound, type, language, emote, comment) VALUES (-1900201, 'I killed another Warmaul ogre! I am Shadow Council! Come get me.', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 0, 0, 0, 0, 'Boulderfist Saboteur');
+INSERT INTO script_texts (entry, content_default, content_loc1, content_loc2, content_loc3, content_loc4, content_loc5, content_loc6, content_loc7, content_loc8, sound, type, language, emote, comment) VALUES (-1900202, 'You make dat sound pretty good. Sound like da orc dieded.', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 0, 0, 0, 0, 'Boulderfist Saboteur');
+INSERT INTO script_texts (entry, content_default, content_loc1, content_loc2, content_loc3, content_loc4, content_loc5, content_loc6, content_loc7, content_loc8, sound, type, language, emote, comment) VALUES (-1900203, 'Dat\'s all of dem! Dese Warmaul be so mad! Hooray!', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 0, 0, 0, 0, 'Boulderfist Saboteur');
+
+-- Make Kil'sorrow Invaders unselectable so they can't be attacked
+UPDATE creature_template SET unit_flags=2 WHERE entry=18397;
```

Comment:
you miss to make the bodies truly dead the aura feign dead.

correct fix would be

```
-- Fix quest Body of Evidence
UPDATE `creature_template` SET `ScriptName` = 'npc_warmaul_pyre' WHERE `entry` = '18395';
UPDATE `creature_template` SET `dynamicflags` = '32' WHERE entry = '18397';
UPDATE `creature_template` SET `flags_extra` = '2' WHERE entry = '18395';

UPDATE  `creature_template_addon` SET  `auras` =  '29266 0 29266 1' WHERE  `creature_template_addon`.`entry` =18397;
```